src/Pact/Types/ExpParser.hs

```diff
@@ -222,7 +222,7 @@ pTokenEpsilon test ps = ParsecT $ \s@(State input o pst _) _ _ eok eerr ->
 -- {-# INLINE pToken #-}
 
 -- | Call commit continuation with current state.
-pCommit :: forall e s m. Ord (Token s) => ParsecT e s m ()
```

Comment:
This one is required by recent versions of megaparsec. It's merged into pact master. However, that patch also includes and module pragma that disables the unused constrained warning for backward compatibility.

https://github.com/kadena-io/pact/blob/0f22b41543a9397391477fb26d1d62798aa23803/src/Pact/Types/ExpParser.hs#L16

Reply:
This is now fixed in pact master and merged into this PR. This change should not be needed any more